plugins/parsers/prometheusremotewrite/parser_test.go

```diff
@@ -40,7 +40,30 @@ func TestParse(t *testing.T) {
 	inoutBytes, err := prompbInput.Marshal()
 	require.NoError(t, err)
 
-	expected := []telegraf.Metric{
+	expectedV1 := []telegraf.Metric{
```

Comment:
Can we please split this test into a `TestParseV1` and `TestParseV2` instead of mangling this stuff?